packages/carbon-react/src/components/Popover/index.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { settings } from 'carbon-components';
```

Comment:
I think this is fine if you want to just remove `packages/react/src/components/Popover/index.js` so the source of truth lives in `@carbon/react`